cli/src/main/java/com/devonfw/tools/ide/tool/IdeasyCommandlet.java

```diff
@@ -137,7 +135,7 @@ public VersionIdentifier getLatestVersion() {
    */
   public boolean checkIfUpdateIsAvailable() {
     VersionIdentifier installedVersion = getInstalledVersion();
-    if (!this.context.isOnline()) {
+    if (this.context.isOffline()) {
       this.context.success("Your version of IDEasy is {}.", installedVersion);
       this.context.warning("Skipping check for newer version of IDEasy due to lack of network connectivity.");
```

Comment:
Suggestion to improve log message.
ATTENTION: Also needs adoption of `StatusCommandletTest.testStatusWhenOffline()`.
```suggestion
      this.context.warning("Skipping check for newer version of IDEasy because you are offline.");
```

Reply:
Implemented together with the adaption of testStatusWhenOffline() in the next commit.